spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -109,6 +109,8 @@
             .add("zeppelin.spark.maxResult",
                 getSystemDefault("ZEPPELIN_SPARK_MAXRESULT", "zeppelin.spark.maxResult", "1000"),
                 "Max number of SparkSQL result to display.")
+            .add("spark.auto.restart.sc", "true", "spark context will automatically restart" +
```

Comment:
If this is a zeppelin config, I think this should start with 'zeppelin', ie, zeppelin.spark.auto.restart.sc